client/src/components/EventCard/ProgrammingLanguages.tsx

```diff
@@ -0,0 +1,58 @@
+import { Box } from "@mui/material";
+import NodeJS from "../../assets/ProgrammingLanguages/nodejs.png";
+import ExtraLanguages from "./ExtraLanguages";
+
+export const thresholdOfLanguages = 4;
+
+export const languages = [
+    "NodeJS",
+]as const;
+
+export type Language = typeof languages[number];
+
+export const images: Record<Language, string> = {
+    "NodeJS": NodeJS,
+}
+
+const ProgrammingLanguage = ({language}:{language: Language}) => {
+    return (
+        <Box
+            component="img"
+            sx={{
+                display: "flex",
+                alignItems: "flex-end",
+                justifyContent: "flex-end",
+                width: "5%",
```

Comment:
Remove all percentages from everywhere.

Reply:
Removed and replaced them with values following the given theme.